CONTRIBUTING.md

```diff
@@ -0,0 +1,223 @@
+# Nodejs Loaders Contributing Guide
+
+Thank you for considering contributing to Nodejs Loaders! We welcome all contributions, whether they are bug reports, feature requests, pull requests, or just questions.
+
+## Getting started
+
+The steps below will give you a general idea of how to prepare your local environment for nodejs loaders development.
+
+1. Click the fork button in the top right to clone the [Node.js Website Repository](https://github.com/nodejs-loaders/nodejs-loaders/fork)
+
+2. Clone your fork using SSH, GitHub CLI, or HTTPS.
+```bash
+git clone git@github.com:<YOUR_GITHUB_USERNAME>/nodejs-loaders.git # SSH
+git clone https://github.com/<YOUR_GITHUB_USERNAME>/nodejs-loaders.git # HTTPS
+gh repo clone <YOUR_GITHUB_USERNAME>/nodejs-loaders.org # GitHub CLI
+```
+
+3. Change into the directory.
+```bash
+cd nodejs-loaders
+```
+
+4. Create a remote to keep your fork and local clone up-to-date.
+```bash
+git remote add upstream git@github.com:nodejs-loaders/nodejs-loaders.git # SSH
+git remote add upstream https://github.com/nodejs-loaders/nodejs-loaders.git # HTTPS
+gh repo sync nodejs-loaders/nodejs-loaders # GitHub CLI
+```
+
+5. Create a new branch for your work.
+```bash
+git checkout -b my-new-feature
+```
+
+6. Install the dependencies.
+```bash
+npm install
+```
+
+7. Run the tests
+```bash
+node --run test
+```
```

Comment:
Maybe add a note that tests for a specific workspace (loader) are possible when using the npm command (`npm run test --workspace=packages/LOADER_NAME`).

Reply:
yeah, maybe mention both